### ISSUE TITLE
REALM_ENABLE_SYNC is not working

### ISSUE THREAD

**Issue by remaerd** [realm/realm-js#844]
Hi,

I come from China and trying Realm with React-Native. It seems like Realm is hosting core and sync iOS library without CDN. I'm suffering with low connection speed with the realm.io server, the download speed is around -40KB/s, which means I cannot finish the download process. I need to download the library with Safari and ShadowSocks (Similar to VPN) to speed things up because curl do not go through ShadowSocks

I discovered that there's a ENABLE_SYNC property in shell code. So I try to disable it. After that, I discovered that I can disable sync function in my Xcode project without modifying your code. So I try it.

But After disabling the feature, Xcode compiler is complaining about unable to find *.hpp in the sync library.

In conclusion, These are my problems:
- The library download connection speed is not steady.
- Lack of documentation about disabling sync.
- Disabling sync is not working.

Wishing those problems can be solved.
Cheers.

**Comment by kristiandupont** [comment]
Hi @remaerd, I'm sorry that you are having download issues, I'll look into that.
As for disabling sync, that is possible but not something we officially support yet. We probably will at some point, but I can't give you a timeline just yet.

**Comment by kristiandupont** [comment]
Apparently, there was an issue with CloudFlare last week. Could you perhaps try downloading again and see if things have improved for you?
